tesseract_command_language/include/tesseract_command_language/profile_dictionary.h

```diff
@@ -156,31 +149,6 @@ class ProfileDictionary
     }
   }
 
-  /**
-   * @brief Check if a profile exists
-   * @details If profile entry does not exist it also returns false
-   * @return True if profile exists, otherwise false
-   */
-  template <typename ProfileType>
-  bool hasProfile(const std::string& ns, const std::string& profile_name) const
```

Comment:
How do you know check if a profile exists?

Reply:
I think this should remain. Using exceptions to check if something exists is very expensive.

Reply:
Note: This will be addressed in profile refactor